### PR TITLE
Add WPML Usage Instructions

### PR DESCRIPTION
**Pull request by emcarru**
Hopefully, WPML will fix this and we can remove this later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +273,23 @@ Flynt includes several utility functions for creating Advanced Custom Fields opt
...
+```php
+<?php
+$autoloadPath = get_template_directory() . '/vendor/autoload.php';
+if (file_exists($autoloadPath)) {
+    require_once $autoloadPath;
+    new \Timber\Timber();
+}
+```
+
+This will ensure Twig is loaded from Timber, not WPML.
```
[the remark this answers is not in the document]
Comment (emcarru) [reply README.md]: You would need to add @domtra for this

[the remark this answers is not in the document]
Comment (domtra) [reply README.md]: @Qakulukiam i haven't gone through everything in wpml to confirm that this will not cause an issue. however, wpml does not update twig in order to keep supporting older php versions.
only features that are deprecated in the twig version that wpml uses were removed in twig 2.0.
i doubt that there will be an issue. event in the wpml forums they say it should be fine to use a newer version of twig. 
we might have problems if twig 3.0 comes out. however, until then we will probably have wp min requirement of php 7.0. 
long story short: this is currently the best way to solve the incompatibilities and i am not aware of any downsides.

File: README.md [hunk 2]
```diff
@@ -271,6 +273,23 @@ Flynt includes several utility functions for creating Advanced Custom Fields opt
 * `Flynt\Utils\Options::addGlobal`<br> Adds fields into a new group inside the Global Options options page. When used with WPML, these fields will always be returned from the primary language. In this way these fields are *global* and cannot be translated.
 * `Flynt\Utils\Options::get` <br> Used to retrieve options from Translatable or Global options.
 
+### WPML
+
+If you are using Flynt and [WPML](https://wpml.org/), you will need to create a Must-Use (MU) Plugin in order to load Twig from Timber before the WPML code is executed. This is because WPML includes an outdated version of Twig. 
+
+To do this, create /wp-content/mu-plugins/flynt.php and add this code:
```
[the remark this answers is not in the document]
Comment (domtra) [reply README.md]: good remark. `timber-compat` or `wpml-compat`?

[the remark this answers is not in the document]
Comment (emcarru) [reply README.md]: My vote would be `flynt-wpml-compat`. Any objections!?

[the remark this answers is not in the document]
Comment (emcarru) [reply README.md]: Done!
